docs/content/operator/common/archives.mdx

```diff
@@ -0,0 +1,172 @@
+
+:::warning
+It is important to prevent pruning until the corresponding data is uploaded. To achieve this, set `use-for-pruning-watermark: true`. This ensures that the node retains the necessary data locally until it has been successfully archived, preventing any loss of important information.
+:::
+
+```yaml
+state-archive-write-config:
+  object-store-config:
+    object-store: "S3"
+    bucket: "<bucket_name>"
+    aws-access-key-id: "<AWS_ACCESS_KEY_ID>"
+    aws-secret-access-key: "<AWS_SECRET_ACCESS_KEY>"
+    aws-region: "<aws_region>"
+    object-store-connection-limit: 20
+  concurrency: 5
+  use-for-pruning-watermark: false
+state-archive-read-config:
+  - object-store-config:
+      object-store: "S3"
+      # Use the same bucket which is being used in `state-archive-write-config`
+      bucket: "<bucket_name>"
+      aws-access-key-id: "<AWS_ACCESS_KEY_ID>"
+      aws-secret-access-key: "<AWS_SECRET_ACCESS_KEY>"
+      aws-region: "<aws_region>"
+      object-store-connection-limit: 20
+    concurrency: 5
+    # This should be set to true in this case. Setting this to true
+    # would prevent pruning of local transaction data until it is archived
+    # in the bucket
+    use-for-pruning-watermark: true
+```
+
+## Set Up an Archival Fallback
+
+To enable your node to fallback to an archive in case of lag, add this to your `fullnode.yaml` file:
+
+<Tabs groupId="object-store">
+<TabItem value="s3" label="Amazon S3">
+
+```yaml
+state-archive-read-config:
+  - object-store-config:
+      object-store: "S3"
+      # Use iota-testnet-archive for testnet 
+      # Use iota-mainnet-archive for mainnet
+      bucket: "iota-<testnet|mainnet>-archive"
+      aws-access-key-id: "<AWS-ACCESS-KEY-ID>"
+      aws-secret-access-key: "<AWS-SECRET-ACCESS-KEY>"
+      aws-region: "<AWS-REGION>"
+      object-store-connection-limit: 20
+    # How many objects to read ahead when catching up  
+    concurrency: 5
+    # Whether to prune local state based on latest checkpoint in archive.
+    # This should stay false for most use cases
+    use-for-pruning-watermark: false
+```
+
+</TabItem>
+<TabItem value="gcs" label="Google Cloud Storage">
+
+```yaml
+state-archive-read-config:
+  - object-store-config:
+      object-store: "GCS"
+      # Use iota-mainnet-archive for mainnet
+      # Notice there is no archive bucket setup for testnet in GCS
+      bucket: "iota-<testnet|mainnet>-archive"
+      google-service-account: "</path/to/service/account/credentials>"
+      object-store-connection-limit: 20
+    # How many objects to read ahead when catching up  
+    concurrency: 5
+    # Whether to prune local state based on latest checkpoint in archive.
+    # This should stay false for most use cases
+    use-for-pruning-watermark: false
+```
+
+</TabItem>
+</Tabs>
+
+
+<!-- ### Configure AWS IAM Permissions for Archive Access
```

Comment:
Is there a reason why this is commented out?

Reply:
We're not providing on AWS now, we'll uncommented it once we have it.